include/nbl/config/BuildConfigOptions.h.in

```diff
@@ -19,6 +19,9 @@
 #cmakedefine _NBL_COMPILE_WITH_OPENGL_
 #cmakedefine _NBL_COMPILE_WITH_VULKAN_
```

Comment:
always compile with all APIs

Reply:
(incl OpenCL)
only CUDA should be optional because the SDK that has the headers is like 2GB and the headers are non redistributable

Reply:
still needs to go